defaults/main.yml

```diff
@@ -26,7 +26,7 @@ schema_registry_root_path: ""
 schema_registry_bin_path: "{{ schema_registry_root_path }}/bin"
 schema_registry_skeleton_paths:
   - "{{ schema_registry_bin_path }}"
-schema_registry_exec_name_start: schema-registry-start
+schema_registry_exec_name_start: find /share/java/schema-registry/ -name \*schema-registry-[0-9]\* -type f  | sed 's/.*schema-registry-\([0-9.]*\)\..*/\1/'
```

Comment:
maybe you should use another var. This one is used in the service template to start it